pkg/apis/extensions/v1beta1/types.go

```diff
@@ -1036,3 +1037,58 @@ type PodSecurityPolicyList struct {
 	// Items is a list of schema objects.
 	Items []PodSecurityPolicy `json:"items"`
 }
+
+// Template contains the inputs needed to produce a Config.
+type Template struct {
+	unversioned.TypeMeta `json:",inline"`
```

Comment:
Yes I will do that before sending for review.  Do you think the kubectl updates should be a separate PR as well?